fiftyone/multimodal/adapters/base.py

```
@@ -0,0 +1,15 @@
+"""
+Base interfaces for multimodal adapters.
+
+| Copyright 2017-2026, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+
+import abc
+
+
+class MultimodalAdapter(abc.ABC):
```

Ruff (0.15.10)

[warning] `MultimodalAdapter` is an abstract base class, but it has no abstract methods or properties

(B024)